ghost/core/test/regression/api/admin/members-signin-url.test.js

```diff
@@ -99,4 +99,60 @@ describe('Members Sigin URL API', function () {
                 .expect(403);
         });
     });
+    describe('With an admin API key', function () {
+        let key, token;
+        before(async function () {
+            await localUtils.startGhost();
+            request = supertest.agent(config.get('url'));
+            await testUtils.initFixtures('members', 'api_keys');
+
+            const admin = await testUtils.createUser({
+                user: testUtils.DataGenerator.forKnex.createUser({email: 'admin+1@ghost.org'}),
+                role: testUtils.DataGenerator.Content.roles[0].name
+            });
+            request.user = admin;
```

Comment:
This doesn't appear necessary. If authenticated via key, no user should be present (I think?).

Reply:
One side note, idk why these are regression tests. That's not a you problem, that's a problem with our patterns an lack of consistency in core :/.

Reply:
IDK either.  I'm honestly not convinced I'm even actually testing much of anything, given that we have fake-fixtures for testing, and I had to change them manually.  But hey, we have tests and they pass, so... yeah?

Reply:
The tests are good 👍. I checked the middleware and it's authenticating via API key as we'd like to see. We're not so worried about testing the migration as the underlying functionality. Agreed we could probably do more to cover that particular piece, but we instead mitigate migration troubles with extra review and a 'simple as possible' approach for them.